gh-repo-export

```diff
@@ -231,10 +256,36 @@ download_migration() {
   fi
 }
 
+print_archive_url() {
+  local MIGRATION_ID="$1"
+  local MIGRATION_STATE="${MIGRATIONS[$MIGRATION_ID]}"
+
+  if [[ $GH_HOST != "" ]]; then
+    if [[ "$GH_HOST" == "github.com" ]]; then
+      BASE_URL="https://api.github.com"
+    else
+      BASE_URL="https://$GH_HOST/api/v3"
+    fi
+  else
+    BASE_URL="https://api.github.com"
+  fi
+
+  if [[ "$MIGRATION_STATE" == "exported" ]]; then
+    local ARCHIVE_URL=$(curl -s -I -H "Authorization: bearer $TOKEN" "$BASE_URL/orgs/$ORGANIZATION/migrations/$MIGRATION_ID/archive" | tr -d '\r' | sed -En 's/^location: (.*)/\1/p')
+    printf "Archive URL for migration %s: %s\n" "$MIGRATION_ID" "$ARCHIVE_URL"
+  else
+    printf "Skipping migration %s due to '%s' state; please investigate\n" "$MIGRATION_ID" "$MIGRATION_STATE"
+  fi
+}
+
 for MIGRATION_ID in "${!MIGRATIONS[@]}"; do
-  if $ARCHIVE_PER_REPO; then
-    download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive}-$MIGRATION_ID.tar.gz"
+  if $SKIP_DOWNLOAD; then
+    print_archive_url "$MIGRATION_ID"
   else
-    download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive-$MIGRATION_ID}.tar.gz"
+    if $ARCHIVE_PER_REPO; then
+      download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive}-$MIGRATION_ID.tar.gz"
+    else
+      download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive-$MIGRATION_ID}.tar.gz"
+    fi
```

Comment:
(nit): totally makes sense but prefer flattening it a little more

```suggestion
for MIGRATION_ID in "${!MIGRATIONS[@]}"; do
  if $SKIP_DOWNLOAD; then
    print_archive_url "$MIGRATION_ID"
  else if $ARCHIVE_PER_REPO; then
    download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive}-$MIGRATION_ID.tar.gz"
  else
    download_migration "$MIGRATION_ID" "${ARCHIVE:-migration-archive-$MIGRATION_ID}.tar.gz"
  fi
```

Reply:
👍🏻